pandas/plotting/api.py

```diff
@@ -0,0 +1,20 @@
+"""
+Plotting api
```

Comment:
I think this is cleaner here actually. And it is only imported explicity (unlike ``__init__`` which is implicity imported)